assets/src/dashboard/app/views/myStories/index.js

```diff
@@ -37,26 +37,66 @@ import {
 } from '../../../components';
 import { STORY_STATUSES } from '../../../constants';
 import { ApiContext } from '../../api/apiProvider';
+import MyStoriesSearch from './myStoriesSearch';
 
+const PageHeading = styled.div`
+  display: flex;
+  justify-content: space-between;
+`;
+
+const SearchContainer = styled.div`
+  position: absolute;
+  right: 20px;
+  display: flex;
+  justify-content: flex-end;
+  margin-top: 40px;
+`;
 const FilterContainer = styled.div`
   padding: 0 20px 20px;
   border-bottom: ${({ theme }) => theme.subNavigationBar.border};
 `;
 
 function MyStories() {
-  const [status, setStatus] = useState(STORY_STATUSES[0].value);
   const {
     actions: { fetchStories },
     state: { stories },
   } = useContext(ApiContext);
 
+  const [status, setStatus] = useState(STORY_STATUSES[0].value);
+  const [typeaheadValue, setTypeaheadValue] = useState('');
+
   useEffect(() => {
     fetchStories({ status });
   }, [fetchStories, status]);
 
+  const filteredStories = useMemo(() => {
+    return stories.filter((story) => {
+      const lowerTypeaheadValue = typeaheadValue.toString().toLowerCase();
+
+      return (
+        story.title.toLowerCase().includes(lowerTypeaheadValue) ||
+        story.id.toString().toLowerCase().includes(lowerTypeaheadValue)
```

Comment:
not really sure this needs to be here since the id is a number but i know we'll be filtering on some other values so i figured this is ok for the time being since the filtering isn't complete yet.

Reply:
Yeah, once we add in the metadata filtering and such, it might be nice to pull this logic out into a separate util file so we can run tests against its ability to filter a set of story data with all sorts metadata properties.

Reply:
💯  yes totally. i'll do this with the filter update to handle metadata so we can get some tests in from the get go if that's cool with you?

Reply:
Super cool with moi!